mpcarolin/re-theme
# Zen 309 fix hook factory

**Ticket**: [ZEN-314](https://jira.simpleviewtools.com/browse/ZEN-314)
**Docker Image** [docker.pkg.github.com/simpleviewinc/keg-packages/keg-components:zen-314-fix-usethemepath](docker.pkg.github.com/simpleviewinc/keg-packages/keg-components:zen-314-fix-usethemepath)

## Context

* The hook factory was returning the wrong value in some cases
  * Also cleaned up a few things in some places, but did change the code
* Added a buildHook, but it's not being used
  * I might use it in the future, so I'm leaving it there for the time being

## Goal
* Fixes an issue where in some cases the onThemeHover / onThemeActive didn't return the correct styles

## Updates
* `src/hooks/hookFactory.js`
  * Fixed return value
* `rollup.config.js`
  * Added build hook even though it's not being used
* `package.json`
  * Added nodemon to auto-restart the server when config files change

## Testing

* Run `yarn install`
* Run `app:install`
* Run `yarn app`
* Ensure the app renders correctly
